### ISSUE TITLE
Is the project alive ?

### ISSUE THREAD

**Issue by Neurozone** [agendav/agendav#271]
All is in the title...

**Comment by ThatComputerKid** [comment]
As far as I can tell, sadly no...

**Comment by pixelbrackets** [comment]
tl;dr Yes

@jorgelzpz and I had a very friendly exchange and he agreed to put the project into a maintenance mode. This means that the project is still alive and will receive commits to keep the software stable and compatible with newer systems. There is no plan to build a mayor version however. I explained the mode in the README, see https://github.com/agendav/agendav#maintenance-mode :construction: :construction_worker: 

As a first effect of this all open PRs for bugfixes have been merged into new releases.

I will triage all open issues next. Everybody is welcome to send pull requests and help to keep this great software up to date.
